icons/filezilla.svg

```diff
@@ -1 +1 @@
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>FileZilla icon</title><path d="M23.999 1.8615V.8028a.8096.8096 0 01-.5678-.236.8097.8097 0 01-.234-.5668h-1.0607c-.021.4439-.3998.7858-.8437.7648A.8066.8066 0 0120.529 0h-1.0607a.8047.8047 0 01-.8527.7548c-.405-.025-.7288-.35-.7548-.7548H16.802a.8072.8072 0 01-.8448.7648c-.4128-.021-.7427-.352-.7647-.7648h-1.0597c-.022.4439-.3989.7858-.8438.7648-.4129-.021-.7428-.352-.7648-.7648h-1.0586a.8047.8047 0 01-.8528.7548C10.208.7298 9.8841.4048 9.8581 0H8.7984c-.011.4439-.3829.7908-.8257.7738A.8034.8034 0 017.1969 0H6.1372c-.021.4439-.3999.7858-.8437.7648-.413-.022-.7438-.352-.7638-.7648H3.47c-.016.4439-.3889.7908-.8318.7738C2.2164.7598 1.8785.4208 1.8635 0H.8038c0 .211-.086.4189-.235.5668a.8096.8096 0 01-.5678.236v1.0587c.4439.017.7898.3888.7738.8317a.8003.8003 0 01-.7738.7748v1.0607a.8026.8026 0 01.7738.8317c-.013.422-.352.7598-.7738.7748V7.194c.4439 0 .8038.3609.8038.8048a.8038.8038 0 01-.8038.8037v1.0587a.8032.8032 0 01.8038.8038.8025.8025 0 01-.8038.8038v1.0596a.8032.8032 0 01.7738.8318c-.014.4219-.353.7598-.7738.7748v1.0607c.4439 0 .8038.3589.8038.8027a.8034.8034 0 01-.8038.8048v1.0607c.4439.016.7898.3869.7738.8297-.014.422-.353.7618-.7738.7748v1.0607a.8034.8034 0 01.8038.8048.803.803 0 01-.8038.8027v1.0607c.4419.002.8018.359.8028.8028h1.0597c.027-.4429.4098-.7808.8537-.7538A.8042.8042 0 013.471 24h1.0607c.021-.4429.3999-.7848.8437-.7638A.8064.8064 0 016.1392 24H7.198c.023-.4429.3999-.7848.8448-.7638A.8064.8064 0 018.8064 24h1.0597c.017-.4429.389-.7908.8318-.7748.4219.016.7608.354.7748.7748h1.0606c.021-.4429.4-.7848.8438-.7638a.8064.8064 0 01.7638.7638h1.0587c.023-.4429.3998-.7848.8447-.7638.4129.021.7428.352.7648.7638h1.0587c.023-.4429.3999-.7848.8447-.7638.413.021.7428.352.7648.7638h1.0597c.016-.4429.3879-.7908.8308-.7748.4218.016.7607.354.7747.7748h1.0547c0-.211.086-.4179.235-.5678a.8079.8079 0 01.5668-.234v-1.0607c-.4439-.027-.7808-.4078-.7548-.8517a.8051.8051 0 01.7548-.7538v-1.0607c-.4439-.023-.7858-.3999-.7638-.8437a.8041.8041 0 01.7638-.7638V16.803c-.4439-.027-.7808-.4099-.7548-.8538.025-.4058.35-.7287.7548-.7537v-1.0607c-.4439-.021-.7858-.4-.7638-.8438.021-.4119.351-.7418.7638-.7638v-1.0576c-.4439-.027-.7808-.41-.7548-.8538.025-.4059.35-.7288.7548-.7538V8.8024c-.4439-.021-.7858-.3999-.7638-.8437a.8056.8056 0 01.7638-.7638V6.1352c-.4439-.027-.7808-.4079-.7548-.8527.025-.405.35-.7288.7548-.7538V3.469c-.4439-.022-.7858-.3999-.7648-.8438a.8058.8058 0 01.7648-.7637zm-4.0668 9.835L13.53 17.9209c.5798.09 1.1396.225 1.7414.225 1.4646 0 3.1471-.445 4.2858-.9158l-.5838 2.7452c-1.6755.8047-2.7002.8707-3.702.8707-1.0946 0-2.1433-.3569-3.214-.3569-.6028 0-1.4736.045-2.0084.4l-1.1607-2.0525 6.605-6.539H7.8928l-1.4506 6.806h-3.169L6.577 3.528h10.487l-.6699 3.145H9.0973l-.6248 2.9242H20.446z"/></svg>
```

Comment:
Looks perfect! Can you reduce down to 3 decimals? 😅

Reply:
Done